packages.md

```diff
@@ -4,6 +4,44 @@ title: Packages
 
 See: [The Complete Idiot's Guide to Common Lisp Packages][guide]
 
+## Using packages
+
+To install and use a package with a single command, use `ql:quickload` as follows:
+~~~lisp
+(ql:quickload :package)
```

Comment:
I think this sentence misleads packages with systems. 

Also quickload is shown on systems.md / loading a system.

and how to access symbols with `package:function` is shown below at "accessing symbols from a package".

The current recipe shows a package definition, that effectively creates a package.  I wouldn't start this section with a quickload example.

Could you rephrase, or add an introduction explaining the difference between systems and packages, linking to the systems page if necessary?

Reply:
Yes - to be honest - I haven't yet grasped the difference between those terms.